dev/MRTCore/azure-pipelines.yml

```diff
@@ -284,112 +284,121 @@ jobs:
     inputs:
       buildType: 'current'
       downloadType: 'single'
-      artifactName: 'binaries_raw'
+      artifactName: 'mrtcore_binaries'
       downloadPath: '$(Build.ArtifactStagingDirectory)\mrt_raw'
 
 # START of copying files into their final package location.
 
   - task: CopyFiles@2
     displayName: 'copy winmd'
     inputs:
-      SourceFolder: '$(Build.ArtifactStagingDirectory)\mrt_raw\binaries_raw\lib\anycpu'
+      SourceFolder: '$(Build.ArtifactStagingDirectory)\mrt_raw\mrtcore_binaries\lib\anycpu'
       Contents: |
         Microsoft.ApplicationModel.Resources.winmd
-      TargetFolder: '$(Build.ArtifactStagingDirectory)\drop\lib\uap10.0\'
+      TargetFolder: '$(Build.ArtifactStagingDirectory)\fatnuget\lib\uap10.0\'
+      flattenFolders: true
+
+  - task: CopyFiles@2
+    displayName: 'copy winmd'
+    inputs:
+      SourceFolder: '$(Build.ArtifactStagingDirectory)\mrt_raw\mrtcore_binaries\lib\anycpu'
+      Contents: |
+        Microsoft.ApplicationModel.Resources.winmd
+      TargetFolder: '$(Build.ArtifactStagingDirectory)\fatnuget\lib\uap10.0\'
       flattenFolders: true
```

Comment:
why do this again?

Reply:
Ah, copy paste error from when I was assembling both the fat nuget and the thin nuget in this file instead of the CreateNuget yaml file.